packages/core/config/storybook/main.js

```diff
@@ -6,7 +6,7 @@ const { getSharedPlugins } = require('../webpack.common')
 
 module.exports = {
   stories: [
-    `${getPaths().web.src}/**/*.stories.{tsx,jsx,js}`.replace(/\\/g, '/'),
+    '../../../../web/src/**/*.stories.{tsx,jsx,js}',
```

Comment:
Is there no way to solve this via improvement to `getPaths()`? 

I'm not sure if this is correct, but I feel like we keep running into various path issues, patching them, and then running into them once again down the road.

Seems soon we need improved util functions that handle all cases and have corresponding tests. @dthyresson Maybe step one is to get this in. Then in a separate PR come back to getPaths() and harden appropriately.

Reply:
Yeah, maybe we should use: https://www.npmjs.com/package/escape-path-with-spaces

Reply:
@thedavidprice > Is there no way to solve this via improvement to getPaths()?

Perhaps. But from what I have seen, paths are often used in different contexts and depending on that context how they are "formed" or escaped (or not) matters.

I initially thought the same -- fix the origin "path" and all downhill will work fine.

But some cases still failed.

One such case is the `DATABASE_URL` used as an env-ar in the Prisma schema. If that has extra " or was escaped, it failed.

Are you suggesting that in the internal paths here:

```
    web: {
      routes,
      base: path.join(BASE_DIR, 'web'),
      pages: path.join(BASE_DIR, PATH_WEB_DIR_PAGES),
      components: path.join(BASE_DIR, PATH_WEB_DIR_COMPONENTS),
      layouts: path.join(BASE_DIR, PATH_WEB_DIR_LAYOUTS),
      src: path.join(BASE_DIR, PATH_WEB_DIR_SRC),
...
```

Instead of `path.join` can a function that `escape-path-with-spaces` and does a path.join()?

Not sure that will help with `processPagesDir` since it also does its own `path.join`s or `ensurePosixPath` that does its own.

Should `ensurePosixPath` really be `escape-path-with-spaces`?

For this PR I will try to use `escape-path-with-spaces` on the Storybook stories config using the existing get paths and see for that fairs.

Reply:
I tried

```
module.exports = {
  stories: [
    `${escape(getPaths().web.src)}/**/*.stories.{tsx,jsx,js}`
  ],
```

and unfortunately, this did not help.

I still got memory errors because of the path:

```
╰───────────────────────────────────────────────────╯
webpack building...
10% building 2/2 modules 0 active
<--- Last few GCs --->

[97886:0x110008000]   241721 ms: Scavenge (reduce) 4027.7 (4100.7) -> 4027.1 (4101.7) MB, 4.2 / 0.0 ms  (average mu = 0.089, current mu = 0.032) allocation failure 
[97886:0x110008000]   241731 ms: Scavenge (reduce) 4027.7 (4100.7) -> 4027.2 (4102.0) MB, 5.7 / 0.0 ms  (average mu = 0.089, current mu = 0.032) allocation failure 


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0x1012b8fd5 node::Abort() (.cold.1) [/Users/dthyresson/.nvm/versions/node/v14.7.0/bin/node]
```

Perhaps because not only do I have a space, but I also have parentheses.

So far the *only* approach that works is to:

* enclose these paths in double quotes as needed 
* relative paths